src/style-spec/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 13.16.0
+
+### ✨ Features and improvements
```

Comment:
Just noticed this also needs to be added here https://github.com/mapbox/mapbox-gl-js/releases/tag/style-spec%4013.16.0

Reply:
Thanks a lot for reminding me that!